project/CopyrightHeader.scala

```diff
@@ -27,8 +28,7 @@ object CopyrightHeader extends AutoPlugin {
         )
       )
     },
-    ValidatePR / additionalTasks += Compile / headerCheck,
-    ValidatePR / additionalTasks += Test / headerCheck
+    Def.settings(additionalTasks += Compile / headerCheck, additionalTasks += Test / headerCheck),
```

Comment:
It might be better to even here be explicit and use `AkkaHttpValidatePullRequest.additionalTasks`